doc/sphinx/references.bib

```diff
@@ -11985,3 +11985,14 @@ @phdthesis{quinquis:2014
   school={Univerzita Karlova, Matematicko-fyzik{\'a}ln{\'i} fakulta},
   address={Praha}
 }
+
+@article{vankeken:etal:2008,
+author = {Keken, Peter and Currie, Claire and King, Scott and Behn, Mark and Cagnioncle, Amandine and He, Jiangheng and Katz, Richard and Lin, Shu-Chuan and Parmentier, E.M. (Marc)},
```

Comment:
You're missing two authors, and the entry for Parmentier is written in a funny way.
```suggestion
author = {Keken, Peter and Currie, Claire and King, Scott and Behn, Mark and Cagnioncle, Amandine and He, Jiangheng and Katz, Richard and Lin, Shu-Chuan and Parmentier, E. Marc and Spiegelman, Marc and Wang, Kelin},
```